### ISSUE TITLE
[Feature Request] A message flag for the submit command

### ISSUE THREAD

**Issue by ZapAnton** [exercism/ideas#13]
Quite often, after submitting an iteration of the exercise, I have to write a small comment on the exercise web page.

Would be nice to have a flag for it, maybe similar to Git (for instance `exercism submit my_solution_file -m "Fixed the memory leak on the line 87"`).

Would it be a worthy feature, or is this case to rare to bother?

**Comment by iHiD** [comment]
Yeah, I like it too.

**Comment by ZapAnton** [comment]
If the feature is to be added what are the necessary steps? Add a flag to the `cli` and expand the submission API to accept the message string?

**Comment by iHiD** [comment]
Yep. If you want to PR to CLI I'm sure that would be appreciated. Be aware that both cli and website have strong review policies and that both can be slow to review. If there is a successfully reviewed pr on cli, I'll do the needful on website.
